config/riscv64/chips/black-parrot/lfs.c

```diff
@@ -0,0 +1,8218 @@
+// BSG Newlib File System Initialization
```

Comment:
Do we have to commit this autogenerated file?

Reply:
So this file was hand generated. Since make isn't run in this directory I don't really know how it would be generated based off of running ./configure from the top level, this is our littlefs interface for newlib file operations so I just included it since all beebs benchmark files use this collection of files (and some of them do stdlib operations)

Reply:
When you say handgenerated, you mean you found all of the input files for beebs and ran bsg_newlib_mkfs on them?

Reply:
If that's the case, it's fine because the other solution (listing out the files, generating it copying it back in) sounds like a pain, and it's not _that_ big.

I worry about this solution for SPEC, whose input sizes are massive.  Although, we'll probably keep the directory on our local server, so it might not be an issue....

Reply:
Oh no sorry I mean I wrote out the command by hand to generate this file system since there is custom makefile. There were no input files for the beebs tests but the libgloss stuff is tightly coupled with an lfs/lfs memory structure as of now so we can't not (yikes double negative lol) have one and still be able to compile/use the current stdlib. For SPEC I will be doing that. I will be preloading all the input text files into a lfs structure (like this one). Yeah I'm not sure about the size :/ I havn't given that a ton of thought to be honest but I can try to keep it in mind

Reply:
Ahh, got it.  Yeah, keep it in mind. Fine for now though